code/datums/objective.dm

```diff
@@ -1078,7 +1078,7 @@ proc/create_fluff(var/datum/mind/target)
 			explanation_text = "Be dastardly as heck!"
 			return
 
-		explanation_text = "Ensure that [target.current.real_name], the [target.assigned_role], safely arrives at Centcom."
+		explanation_text = "Ensure that [target.current.real_name], the [target.assigned_role], safely arrives at Centcom alive and not as a cyborg."
```

Comment:
```suggestion
		explanation_text = "Ensure that [target.current.real_name], the [target.assigned_role], safely arrives at Centcom alive and not as a cyborg/AI."
```
this can occur as well :)